contracts/evoting/controller/action.go

```diff
@@ -240,97 +248,26 @@ func (a *scenarioTestAction) Execute(ctx node.Context) error {
 
 	// ###################################### CREATE SIMPLE ELECTION ######
 
-	fmt.Fprintln(ctx.Out, "Create election")
-
-	// Define the configuration
-	configuration := fake.BasicConfiguration
-
-	createSimpleElectionRequest := ptypes.CreateElectionRequest{
-		Configuration: configuration,
-		AdminID:       "adminId",
-	}
-
-	signed, err := createSignedRequest(secret, createSimpleElectionRequest)
+	err,electionID,election,electionIDBuf :=setupSimpleElection(ctx,secret,proxyAddr1,serdecontext,electionFac,service)
```

Comment:
It also increases readibility to have spaces between operators and comas. Make sure that your line are not too long either. We usually want to keep the line between 80 to 100 characters long. You can set visual aids in your IDE to help you realise how long the line is.

```suggestion
	electionID, election, electionIDBuf, err := setupSimpleElection(ctx , secret, proxyAddr1,  serdecontext, electionFac, service)
```